src/name/EnterName.js

```diff
@@ -1,36 +1,41 @@
-import React, {useState} from "react";
+import React, { useState } from "react";
 
 import isValid from "../validates/isValid";
 import Error from "../validates/error";
 
-const EnterName = ({name, setPetName}) => {
-    const [letters, setLetters] = useState(name);
-    const [error, setError] = useState(false);
-
-    const handleSubmit = (e) => {
-        e.preventDefault();
-        if (isValid(letters) && name !== " ") {
-            setError(false);
-            setPetName(letters);
-        } else {
-            setError(true);
-        }
-    }
-
-    const handleChange = (e) => {
-        setLetters(e.target.value);
+const EnterName = ({ name, setPetName }) => {
+  const [letters, setLetters] = useState(name);
+  const [error, setError] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (isValid(letters) && name !== " ") {
```

Comment:
you are checking only for one space, but what if I enter multiple, like `" multiple spaces   "`?

Reply:
the program will trim multiple spaces to one